client/scss/utilities/_all_compiled_variables.scss

```diff
@@ -0,0 +1,4 @@
+@import 'compiled_variables/colors';
+@import 'compiled_variables/container_padding';
+@import 'compiled_variables/gutter_width';
+@import 'compiled_variables/grid_config';
```

Comment:
[My question above](https://github.com/wellcometrust/wellcomecollection.org/pull/1075/files#r124194049) referring to here - if that step hasn't been run - this fails.